src/main/java/frc/robot/RobotContainer.java

```diff
@@ -689,11 +664,19 @@ public void robotPeriodic() {
     public void disabledPeriodic() {
         /* set to coast mode when circuit open */
         if (brakeSwitch != null && brakeSwitch.get()) {
-            scoringSubsystem.setBrakeMode(false);
-            endgameSubsystem.setBrakeMode(false);
+            if (FeatureFlags.runScoring) {
+                scoringSubsystem.setBrakeMode(false);
+            }
+            if (FeatureFlags.runEndgame) {
+                endgameSubsystem.setBrakeMode(false);
```

Comment:
Is the reason this is not working because we are setting the left endgame motor to brake/coast, but it is following the right motor? Could we try to fix this